packages/flutter/test/material/text_field_test.dart

```diff
@@ -2358,6 +2358,7 @@ void main() {
 
   testWidgets('Can move cursor when dragging, when tap is on collapsed selection (iOS)', (WidgetTester tester) async {
     final TextEditingController controller = _textEditingController();
+    addTearDown(controller.dispose);
```

Comment:
I think the method `_textEditingController()` already does this right?

Reply:
Oh, you are right. My bad. Corrected.